src/app/js/lib/propTypes.js

```diff
@@ -8,7 +8,8 @@ export const polyglot = PropTypes.shape({
     tm: PropTypes.func.isRequired,
 });
 
-export const datasetCharacteristic = PropTypes.shape({
+export const property = PropTypes.shape({
```

Comment:
The `DatasetCharacteristicsItem` component still references `datasetCharacteristic` for its propTypes